### PR TITLE
RH 2023-05-31 questions

### PR DESCRIPTION
**Pull request by ralfhandl**

### CODE REVIEW

File: odata-data-aggregation-ext/5 Vocabulary for Data Aggregation.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ The following terms are defined in the vocabulary for data aggregation [OData-Vo
 
 ## ##subsec Aggregation Capabilities
 
-The term `ApplySupported` can be applied to an entity set, entity type or to a collection rooted in an entity container. It describes the aggregation capabilities of the annotated target. If present, it implies that instances of the annotated target can contain dynamic properties as an effect of `$apply` even if they do not specify the OpenType attribute, see [OData-CSDL](#ODataCSDL). The term has a complex type with the following properties:
+The term `ApplySupported` can be applied to an entity set, entity type or to a collection rooted in an entity container. <!-- Where do we define "rooted in an entity container"? --> It describes the aggregation capabilities of the annotated target. If present, it implies that instances of the annotated target can contain dynamic properties as an effect of `$apply` even if they do not specify the OpenType attribute, see [OData-CSDL](#ODataCSDL). The term has a complex type with the following properties:
```
Comment (HeikoTheissen) [inline odata-data-aggregation-ext/5 Vocabulary for Data Aggregation.md:9]: https://github.com/standardapi/odata-specs/pull/13#discussion_r1211180626

File: odata-data-aggregation-ext/5 Vocabulary for Data Aggregation.md [hunk 2]
```diff
@@ -179,6 +179,8 @@ A node without parent node is a _root node_, a node is a _child node_ of its par
 
 The term `UpNode` can be used in hierarchical result sets to associate with each instance one of its ancestors, which is again annotated with `UpNode` and so on until a path to the root is constructed.
 
+<!-- TODO: add UpNode to Aggregation vocabulary for CS03 -->
```
Comment (HeikoTheissen) [inline odata-data-aggregation-ext/5 Vocabulary for Data Aggregation.md:182]: https://github.com/oasis-tcs/odata-vocabularies/blob/e69a1bf5b1d9ada497e5e8f271bdabe5f0589c19/vocabularies/Org.OData.Aggregation.V1.xml#L516-L524

File: odata-data-aggregation-ext/5 Vocabulary for Data Aggregation.md [hunk 3]
```diff
@@ -369,3 +371,5 @@ results in
 :::
 
 Further examples for recursive hierarchies using transformations operating on the hierarchy structure are provided in [Aggregation in Recursive Hierarchies](#AggregationinRecursiveHierarchies).
+
+<!-- What about term AvailableOnAggregates: do we want to deprecate it? -->
```
Comment (HeikoTheissen) [inline odata-data-aggregation-ext/5 Vocabulary for Data Aggregation.md:375]: Still needed for functions that serve as service-specific set transformations.

File: odata-data-aggregation-ext/3.3 Transformations Preserving the Input Set Structure.md [hunk 4]
```diff
@@ -114,6 +116,8 @@ results in
 
 The first parameter MUST evaluate to a number $s$. The second parameter MUST be an [aggregatable expression](#AggregatableExpression) that evaluates to a number. In step 5, exit the loop if the sum of the numbers addressed by the second parameter in the output set is greater than or equal to a non-negative $s$ or is less than or equal to a negative $s$.
 
+<!-- So bottomsum(-2, Amount) with Amount in {-1, 0, 1} returns all three input set entries? This is surprising, and an explanation plus an example would be helpful. -->
```
[the remark this answers is not in the document]
Comment (ralfhandl) [reply odata-data-aggregation-ext/3.3 Transformations Preserving the Input Set Structure.md:119]: So it's not "bottom", but "close to zero"?

File: odata-data-aggregation-ext/3.3 Transformations Preserving the Input Set Structure.md [hunk 5]
```diff
@@ -12,6 +12,8 @@ GET /service/Sales?$apply=
 where the number in `skip` is $M=(i-1)⋅N$.
 :::
 
+<!-- Does this restriction on M mean that one can't skip half pages? Or can i be a rational number? -->
```
[the remark this answers is not in the document]
Comment (HeikoTheissen) [reply odata-data-aggregation-ext/3.3 Transformations Preserving the Input Set Structure.md:15]: https://github.com/standardapi/odata-specs/blob/95f442298f515e8e0e4c9de77c8a30714e44992a/odata-data-aggregation-ext/3.3%20Transformations%20Preserving%20the%20Input%20Set%20Structure.md?plain=1#L6

File: odata-data-aggregation-ext/3.3 Transformations Preserving the Input Set Structure.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (HeikoTheissen) [reply odata-data-aggregation-ext/3.3 Transformations Preserving the Input Set Structure.md:119]: https://github.com/standardapi/odata-specs/pull/10/files#r1211260864

**Comment by HeikoTheissen** [comment]
Open questions moved to #10, suggest to close this.

File: odata-data-aggregation-ext/3.3 Transformations Preserving the Input Set Structure.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (ralfhandl) [reply odata-data-aggregation-ext/3.3 Transformations Preserving the Input Set Structure.md:15]: The problematic text "where the number in `skip` is $M=(i-1)⋅N$." is still there: did I miss something?

[the remark this answers is not in the document]
Comment (HeikoTheissen) [reply odata-data-aggregation-ext/3.3 Transformations Preserving the Input Set Structure.md:15]: https://github.com/standardapi/odata-specs/pull/10/commits/5575a6996e3e6096b34982fa78559d92924b384f

**Pull request closed without merging**